_pages/learn.md

```diff
@@ -3,14 +3,53 @@ title: Learn research software engineering
 author_profile: false
 permalink: /learn/
 layout: single
+toc: true
 ---
 
-Do you want to become an RSE? Are you looking to obtain new [competencies]({{ site.baseurl }}/competencies)? We recommend starting from these resources.
+Do you want to become an RSE? While there are few (if any) RSE-specific study programs at the moment, many are, at their core, programs that teach RSE skills. But, even if you have already studied something else, several online resources may help you obtain new [competencies]({{ site.baseurl }}/competencies).
+
+## Study programs
+
+### Curricula on Scientific Computing
+
+| Level | Name            | University | Country | Language |
+| ---   | --------------- | ---------- | ------- | -------- |
+| B.Sc. | [Computing in Science](https://www.uni-hamburg.de/campuscenter/studienangebot/studiengang.html?1239877544)                                                                                       | University of Hamburg                 | Germany | German  |
```

Comment:
Would it make sense to store the information in this table in e.g. YAML, and generate the table with a Liquid loop? This would facilitate data re-use by third-party actors, as YAML files are easier to parse than tabular data, and would also make the data more human-readable. Right now the lines are very long (to preserve column alignment in a teletype font), and any new URL that is longer would force us to update all rows.

Reply:
These are exactly my comments in the PR description. :sweat_smile: 
I don't think we need that yet.
Columns: I am happy to switch to non-aligned.

Reply:
Ah sorry, I didn't pay attention. Don't bother changing the alignment. :sweat_smile: